### PR TITLE
Add highlight to event card when time come

### PR DESCRIPTION
**Pull request by XiaoXuxxxx**
My time has come

### CODE REVIEW

**Comment by XiaoXuxxxx** [comment]
it's work. trust me bro

File: apps/registration/src/components/HorizontalTimeline.tsx [hunk 1]
```diff
@@ -80,7 +80,7 @@ const Card = ({
   endTime: string,
   small?: boolean,
 }) => {
-  const isCurrentEvent = false; // TODO: WTF TIME ALGO
+  const isCurrentEvent = new Date(startTime) <= new Date() && new Date() <= new Date(endTime);
```
Comment (riflowth) [inline apps/registration/src/components/HorizontalTimeline.tsx:83]: `new Date()` should be initiated once

**Review by riflowth — CHANGES_REQUESTED** [review]

**Comment by riflowth** [comment]
@XiaoXuxxxx Should we update the highlight realtime?

File: apps/registration/src/components/HorizontalTimeline.tsx (hunk 1 above)
Comment (XiaoXuxxxx) [reply apps/registration/src/components/HorizontalTimeline.tsx]: done bro

Comment (XiaoXuxxxx) [reply apps/registration/src/components/HorizontalTimeline.tsx]: Forget to resolve, lol

**Review by riflowth — APPROVED** [review]

**Pull request merged**